core/controllers/admin.py

```diff
@@ -408,8 +408,8 @@ def _load_dummy_new_structures_data(self):
             subtopic_page = (
                 subtopic_page_domain.SubtopicPage.create_default_subtopic_page(
                     1, topic_id_1))
-            self._reload_exploration('0')
-            self._reload_exploration('16')
+            self._reload_exploration('15')
```

Comment:
Why did both these values get changed?

Reply:
Since we are publishing the stories here, the explorations in them had to be valid. Both explorations '0' and '16' failed validations, so I used '15' and '25' as they are valid and changed their categories so they match.

Reply:
Ah ok. Thanks! Maybe add a comment to say "we chose these demo explorations because they pass validation" for the benefit of future developers.

Reply:
Done